testsuite/oslc-warn-commainit/test.osl

```diff
@@ -0,0 +1,5 @@
+shader test ( output color Cout = 0 )
+{
+    vector x = (1, 2, 3);
```

Comment:
Specifically, what it currently says is:

    test.osl:4: error: Can't use '= {...}' initializer except with arrays or struct (x)